store/event_history.go

```diff
@@ -83,12 +83,12 @@ func (eh *EventHistory) scan(key string, recursive bool, index uint64) (*Event,
 
 			if recursive {
 				// add tailing slash
-				key = path.Clean(key)
-				if key[len(key)-1] != '/' {
-					key = key + "/"
+				nkey := path.Clean(key)
```

Comment:
yes, but i do not really want to touch/change v2 related code in any significant way... so if this works, then great...